.travis.yml

```diff
@@ -9,10 +9,10 @@ install:
   # get coveralls.io support
   - go get github.com/mattn/goveralls
   # gometalinter is deprecated; migrating to golangci-lint. See https://github.com/alecthomas/gometalinter/issues/590.
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
-
+  # - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
```

Comment:
Let's create an issue to track this so we don't forget to turn this back on later